docs/architecture/grpc-for-wcf-developers/wcf-bindings.md

```diff
@@ -1,11 +1,35 @@
+
+## HTTP
+
+WCF is only cross-platform when using SOAP over HTTP/1.1. WCF HTTP binding is text based, uses SOAP as the wire format, and is very slow by the standards of modern networked applications. It's only really used to provide cross-platform interoperability, or connection over internet infrastructure. The equivalent in gRPC—because it uses HTTP/2 as the underlying transport layer with the binary Protobuf wire format for messages—can offer Net TCP service level performance but with full cross-platform interoperability with all modern programming languages and frameworks.
+
+## Named Pipes
+
+WCF provided a Named Pipes binding for communication between processes on the same physical machine. Named Pipes is a proprietary Windows feature and doesn’t work like TCP, so HTTP/2 doesn’t work over Named Pipes, meaning gRPC can’t work over Named Pipes.
+
+Outside Windows, the functionality provided by Named Pipes is instead generally provided by Unix Domain Sockets. These sockets are regular TCP-like sockets represented with file-system addresses, such as `/var/run/docker.sock`, which gRPC can work with as both client and server. If you need to use Named Pipes style functionality on Windows, the next update to Windows 10 and Windows Server, in 2019 Q4, adds Domain Sockets as a fully supported native feature within Windows. So, gRPC services running on these and later versions of Windows (or on Linux) can use Domain Sockets instead of Named Pipes. However, if your team is unable to update to the latest version of Windows, then you'll need to use localhost TCP sockets. Security concerns about using local TCP sockets can be addressed with the use of certificate authentication between client and server.
+
+## MSMQ
+
+MSMQ is a proprietary Windows message queue. WCF's binding to MSMQ enables "fire and forget" requests from clients that may be processed at any time in the future. gRPC doesn't natively provide any message queue functionality. The best alternative would be to directly use a messaging system like Azure Service Bus, RabbitMQ, or Kafka. This could be implemented with the client placing messages directly onto the queue, or a gRPC client streaming service that enqueues the messages.
+
+## WCF REST
+
+The WCF REST extensions, such as `WebGet` or `WebInvoke`, enabled you to develop RESTful APIs that could speak JSON at a time when this was less common than now. Therefore, if you have a RESTful API built with WCF REST, consider migrating it to a regular ASP.NET Core MVC Web API application, which would provide equivalent functionality, rather than converting to gRPC.
```

Comment:
The WCF REST extension can be summed up as the WebHttpBinding which supports the WebGet and WebInvoke attributes. You should at least mention this binding as someone reading this might be aware they use WebHttpBinding but not connect the dots to WebGet or WebInvoke. Whenever we're asked for this, WebHttpBinding is what's usually asked for, not WebGet or WebInvoke.